packages/common/src/directives/ng_for_of.ts

```diff
@@ -11,7 +11,7 @@ import {Directive, DoCheck, EmbeddedViewRef, Input, IterableChangeRecord, Iterab
 /**
  * @publicApi
  */
-export class NgForOfContext<T, U extends NgIterable<T>> {
+export class NgForOfContext<T, U extends NgIterable<T> = any> {
```

Comment:
We should be able to use `NgIterable<T>` as default type, here and in all other places:

```suggestion
export class NgForOfContext<T, U extends NgIterable<T> = NgIterable<T>> {
```

Reply:
Done.